dasharo-security/secure-boot.robot

```diff
@@ -233,6 +233,29 @@ SBO008.001 Attempt to enroll the key in the incorrect format (firmware)
     Select File In File Explorer    DB.txt
     Read From Terminal Until    ERROR: Unsupported file type!
 
+SBO009.001 Attempt to boot file signed for intermediate certificate
+    [Documentation]    This test verifies that a file signed with an
```

Comment:
@WiktorG351 Could you elaborate on the description here about how you generated the chain of certs and then how signed the file?

Reply:
I want to be sure that we are really testing what is stated in the name of test.

Reply:
Ok, here we create intermediate certificate by signing csr using PK priv key.